content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -92,23 +90,23 @@ with Live(save_dvc_exp=True) as live:
 <admon>
 
 Using `save_dvc_exp=True` will ensure that
-[the results get saved as a DVC experiment even if you do not have a DVC pipeline](/doc/dvclive/how-it-works#track-the-results).
+[the results get saved as a DVC experiment](/doc/dvclive/how-it-works#track-the-results).
 
 </admon>
 
-<admon>
+<admon type="tip">
 
 DVCLive signals the end of the experiment using `live.end()`. Using
-`with Live() as live:` or one of the DVCLive integrations for
+`with Live() as live:` or one of the integrations for
 [ML Frameworks](/doc/dvclive/api-reference/ml-frameworks) ensures that
-`live.end()` is automatically called when the experiment concludes.
+`live.end()` is automatically called when the experiment concludes successfully.
 
 </admon>
 
 Iterative Studio stores the live metrics and plots data in its database.
 
-In the project table, the live metrics are displayed nested under the parent Git
-commit. Updates to the live metrics are highlighted in orange.
+In the project table, the live metrics are displayed in experiment rows, nested
+under the parent Git commit.
 
 ![](https://static.iterative.ai/img/studio/live_metrics.gif)
```

Comment:
Is this right? The rows under the parent commit are experiments right? I see an exp name in the sample img.

Reply:
Right. But I didn't get your question.

Reply:
I guess I was confused by the wording. I would add "...experiment rows, **which** are nested..."

Reply:
Fixed in https://github.com/iterative/dvc.org/pull/4370